src/api/app/views/webui/attribute/responsive_ux/_index_actions.html.haml

```diff
@@ -0,0 +1,5 @@
+- content_for :actions do
+  %li.nav-item
+  = link_to(new_attribs_path(project: project.name, package: package), title: 'Add attribute') do
+    %i.fas.fa-plus-circle.fa-lg.mr-2
+    Add attribute
```

Comment:
`attribute` should be capitalized, so it would be `Add Attribute` (same for the title on line 3).

Reply:
True! Updated.